x-pack/plugins/cases/public/components/case_action_bar/index.tsx

```diff
@@ -172,6 +172,7 @@ const CaseActionBarComponent: React.FC<CaseActionBarProps> = ({
                     </EuiButtonEmpty>
                   </span>
                 </EuiFlexItem>
+                {/* currently just delete should we limit it to that? */}
```

Comment:
@cnasikas I believe this is for the three dots menu, which at the moment only includes a deletion icon. So technically the user doesn't need `all` to use the menu. I could see additional actions being added that require more than delete in the future though. So maybe we keep it as `all`? It was essentially `all` before.